test/e2e/node/pod_resize.go

```diff
@@ -1408,14 +1441,133 @@ func doPodResizeErrorTests() {
 	}
 }
 
-var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
-	featureGatePostAlpha = true
-	if fs, found := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()[features.InPlacePodVerticalScaling]; found {
-		if fs.PreRelease == featuregate.Alpha {
-			featureGatePostAlpha = false
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-testcase1", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		if nodeNum > 1 {
+			framework.Logf("There are more than 1 node in the testing cluster, so skip the pod-in-queue-resize-test!")
+			return
 		}
-	}
+
+		ginkgo.By("Checking the node allocatable CPU resources!")
+		node := nodes.Items[0]
+		nodeAllocatableResources := node.Status.Allocatable
+		nodeAllocatableCPU := nodeAllocatableResources.Cpu()
+		nodeAllocatableCPUValue := nodeAllocatableCPU.MilliValue()
+		nodeAllocatableCPUStr := nodeAllocatableCPU.String()
+		framework.Logf("Node allocatable CPU is %s", nodeAllocatableCPUStr)
+		testPodRequest := nodeAllocatableCPUValue / 2
+		testPodRequestStr := strconv.FormatInt(testPodRequest, 10) + "m"
+		framework.Logf("Test Pod Request CPU is %s", testPodRequestStr)
+		testPodRequestAfterResize := testPodRequest / 2
+		testPodRequestAfterResizeStr := strconv.FormatInt(testPodRequestAfterResize, 10) + "m"
+		framework.Logf("Test Pod CPU Request After Resize is %s", testPodRequestAfterResizeStr)
+		testPodRequestToMakeSpace := testPodRequestAfterResize / 2
+		testPodRequestToMakeSpaceStr := strconv.FormatInt(testPodRequestToMakeSpace, 10) + "m"
+		framework.Logf("Test Pod CPU Request To Make Space is %s", testPodRequestToMakeSpaceStr)
+
+		ginkgo.By("Making testPod1 that fits the node, testPod2 that cannot fit!")
+		c1 := []TestContainerInfo{
+			{
+				Name:      "c1",
+				Resources: &ContainerResources{CPUReq: testPodRequestStr, CPULim: testPodRequestStr},
+			},
+		}
+
+		c2 := []TestContainerInfo{
+			{
+				Name:      "c2",
+				Resources: &ContainerResources{CPUReq: nodeAllocatableCPUStr, CPULim: nodeAllocatableCPUStr},
+			},
+		}
+
+		patchC2ToFit := fmt.Sprintf(`{
+			"spec":{
+					"containers":[
```

Comment:
not: formatting.. please use space after ':' and alignment. I think if run, ./hack/update-gofmt.sh , that will fix it for you.
```go
			"spec":{
					"containers": [
						{
							"name":         "c2",
							"resources": {"requests":{"cpu":"%s"},"limits":{"cpu":"%s"}}
						}] 
					}
			}`,
```